### DISCUSSION TITLE
Special Characters in “tags” Plugin

### DISCUSSION THREAD

**Discussion by prtst33** [mecha-cms/mecha#237] [Q&A]
Hi, 
How can you make tags plugin display special characters in tags like ā, ē, ī,...etc? 
At the moment it doesn't work.

Thanks.

**Comment by taufik-nurrohman** [comment]
Are you referring to the tags input in panel?

**Reply by prtst33** [reply]
Yes. When I put a special character in the tags section in panel, I save the article and when I post it the special character does not appear.

For example, in the tag form I put the word "naptári" and i save, and on the page it displays the word "naptri".
Thanks.

**Reply by taufik-nurrohman** [reply]
The value you have entered in the tags field will be used as the file name so it&rsquo;s characters must be limited to certain range. You can still change the display title later via the Tag menu in the top-left corner.

![2022-09-07-061831_1366x768_scrot](https://user-images.githubusercontent.com/1669261/188756871-778af204-bd19-44f5-80c7-e9e3cf6b8700.png)
